src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -2597,7 +2598,7 @@ gregtech.machine.block_breaker.lv.name=Basic Block Breaker
 gregtech.machine.block_breaker.mv.name=Advanced Block Breaker
 gregtech.machine.block_breaker.hv.name=Advanced Block Breaker II
 gregtech.machine.block_breaker.ev.name=Advanced Block Breaker III
-gregtech.machine.block_breaker.tooltip=Mines block on front face and collects its drops
+gregtech.machine.block_breaker.tooltip=Mines block on front face and collects its drops/nOnly works when powered with Redstone
```

Comment:
Can you add this to the item collector as well? We found this was also the case for that one.

Reply:
That seems to already be in the tooltip
![2021-12-29_17 58 41](https://user-images.githubusercontent.com/66188216/147709324-a03c5319-2c45-4fae-8be4-a13f81f57aeb.png)